boms/cloud-oss-bom/pom.xml

```diff
@@ -47,21 +47,21 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <guava.version>31.0.1-jre</guava.version>
+    <guava.version>31.1-jre</guava.version>
     <gson.version>2.9.0</gson.version>
-    <google.cloud.bom.version>0.171.0</google.cloud.bom.version>
-    <google.cloud.core.version>2.5.6</google.cloud.core.version>
-    <io.grpc.version>1.44.1</io.grpc.version>
-    <http.version>1.41.4</http.version>
+    <google.cloud.bom.version>0.172.0</google.cloud.bom.version>
+    <google.cloud.core.version>2.5.11</google.cloud.core.version>
+    <io.grpc.version>1.45.0</io.grpc.version>
+    <http.version>1.41.5</http.version>
```

Comment:
In addition to the Google Cloud BOM 0.172.0, these changes come from the shared dependencies BOM 2.9.0
https://github.com/googleapis/java-shared-dependencies/blob/v2.9.0/first-party-dependencies/pom.xml